content/components/page-content/step-indicator.md

```diff
@@ -0,0 +1,183 @@
+---
+title: "Step Indicator"
+asOf: 1.6.0
+description: "Show the user the current step in a multi-step process"
+status: "Beta"
```

Comment:
Let's set this to "Alpha" since its the initial release.

Reply:
My bad, copy/paste error. Fixing now!